packages/gatsby-image/README.md

```diff
@@ -176,8 +176,8 @@ Importing from `gatsby-image/withIEPolyfill` tells Gatsby to automatically apply
 
 There are two types of responsive images supported by gatsby-image.
 
-1. Images that have a _fixed_ width and height
-2. Images that stretch across a _fluid_ container
+1. Images that have a _fixed_ width and height.
+2. Images that stretch across a _fluid_ container.
```

Comment:
I'm on board with correcting punctuation in general but periods should only be added to the end of a list item if the text in that list item is a complete sentence.

Reply:
oh, thanks for the enlightenment I really appreciate it.